actors/market/tests/publish_storage_deals_failures.rs

```diff
@@ -203,4 +208,268 @@ mod publish_storage_deals_failures {
         };
         assert_deal_failure(true, f, ExitCode::USR_ILLEGAL_ARGUMENT, Ok(()));
     }
+
+    #[test]
+    fn fail_when_client_has_some_funds_but_not_enough_for_a_deal() {
+        let mut rt = setup();
+
+        //
```

Comment:
Thanks for diligently copying.  In this case the go test is being sloppy and this comment can go.